src/apply.jl

```diff
@@ -112,3 +112,34 @@ function apply!(table::T, t::Transform; cols=nothing, kwargs...)::T where T
 
     return table
 end
+
+"""
+    apply!!(A::AbstractArray, ::Transform; dims=1, kwargs...)
+
+Applies the [`Transform`](@ref) to each element of `A` and appends the result to `A`, if
+possible, creating a new array.
+"""
+#TODO: syntax would imply this is mutating. Should we make it so a new array isn't constructed?
+function apply!!(data::AbstractArray, t; dims, inds=Colon(), kwargs...)::AbstractArray
+    output = apply(data, t; dims=dims, inds=inds, kwargs...)
+    new_size = collect(size(data))
```

Comment:
```suggestion
    append_size = collect(size(data))
```
I was confused thinking this was the new size of the whole output